src/app/layout/navbar/language-button/language-button.component.html

```diff
@@ -0,0 +1,3 @@
+<a (click)="changeLanguage()">
+  {{language.currentLanguage | uppercase}}
```

Comment:
Be consistent. If bindings will have spaces at the beginning and the end in some places, keep that behavior across all the application.